src/test/java/core/basesyntax/FileReaderImplTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String NON_EXISTING_FILE = "nonexisting.csv";
+    private static final String TRANSACTIONS_FILE = "src/test/resources/inputs/transaction.csv";
+    private static final String TEST_FILE = "src/test/resources/inputs/test.csv";
+    private static final String EMPTY_FILE = "src/test/resources/inputs/empty.csv";
+
+    private FileReader fileReader;
+
+    @Before
+    public void setup() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFile_fromNonExistingFile_notOk() {
+        fileReader.readFile(NON_EXISTING_FILE);
+    }
+
+    @Test
+    public void readFile_ok() {
```

Comment:
Not the full name of the method, you have to stick to that naming:
<img width="343" alt="Screen Shot 2022-08-28 at 6 03 17 PM" src="https://user-images.githubusercontent.com/90692308/187080761-85dcb9e6-1e58-46ac-bfe2-194ac23198dd.png">